README.md

```diff
@@ -135,6 +135,35 @@ the appropriate data, then exporting the fixture:
 ./scripts/manage.sh dumpdata survey.Blockface > src/nyc_trees/apps/survey/fixtures/blockface.json
 ```
 
+When all 5 boroughs have been added, the resulting fixture is too
+large for the ``loaddata`` command. The fixture needs to be split into
+chunks. This process is not fully automated.
+
+This pipline will take a fixture created with ``dumpdata`` on stdin
+and output one object per line.
+
+```
+# convert a JSON array to linewise objects with trailing commas
+# embedded newline is IMPORTANT
+# sed -e 's/ *$//' trims whitespace
+sed -e 's/{"fields/\
```

Comment:
Going to mess with an alternative to this, just because the literal newline seems like it could cause issues, but not going to let that hold things up.